plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -188,7 +188,7 @@ export default class NodesTable extends React.Component<
         width={hasCustomWidth("host") ? customWidthFor("host") : undefined}
       />,
 
-      withMaintenanceStatus ? (
+      (Config.features || {}).maintenance ? (
```

Comment:
What does the `|| {}` do? Wouldn't `{}.mainenence` give an error?

Reply:
no, it would give you `undefined`. That's a safeguard for the case that a config has no `features`-prop, which i assume might be common.